Gemfile

```diff
@@ -87,7 +87,7 @@ gem 'pg', '~> 0.18.0', '< 0.19.0'
 gem 'acts_as_versioned', :git => 'https://github.com/technoweenie/acts_as_versioned.git', :ref => '63b1fc8529d028'
 gem 'active_model_otp', :git => 'https://github.com/heapsource/active_model_otp.git', :ref => '55d93a3979'
 gem 'cancancan', '~> 1.12.0', '< 1.13.0'
-gem 'charlock_holmes', '~> 0.7.0'
+gem 'charlock_holmes', '~> 0.7.0', '< 0.7.4'
```

Comment:
Link to https://github.com/brianmario/charlock_holmes/commit/6fac2094b88b95b6e22b3ce0b4d01cf8743b4ddb in the commit message. When we so come to unpin this, being specific really helps with understanding what the blockers were.